### PR TITLE
Correct some CSS issues remaining on UI

### PR DESCRIPTION
**Pull request by Nilvus**
A new set of CSS improvements. This PR should resolve some remaining issues :

- #2428 : @edgardoh, see if it's ok for you now ? As items in preferences (entries, checkbox, labels) are all on a grid, size is only adjusted by margins between items. So here it's not possible to adjust each one separately. This seems the better compromise on my opinion.
- #2549 : @kanyck, see if it's now ok for you ? It's possible to reduce a little more but that would be too condensed. Have an aerate UI is also great, so I try to find also the better compromise here between aerate text and not too coarse history.

I also reduce size on some dropdown button arrows, especially on filter collection (I also reduce height size on entry item near this dropdown) and display dropdown on filmic that was way too big.

@aurelienpierre : could you see if it's ok for you as it change some of your choices ?

Before merging that, please test as much as you can. I made lot of reviews tests and compare with actual themes what change and if this new css breaks other part. I don't find anything on lot of modules I test, even on map and print parts (I don't test capture part as I don't have a camera compatible).

As I change lot of things, I want to be sure that nothing is broken on the UI.

### CODE REVIEW

File: data/themes/darktable.css [hunk 1]
```diff
@@ -648,7 +666,7 @@ combobox button
   border: 1px solid transparent;
   padding: 1px;
   background-color: transparent;
-  border-radius: 2pt;
+  border-radius: 2px;
```
Comment (TurboGit) [inline data/themes/darktable.css]: I don't think we want PX here. As far as I know PT is better as it translate properly on HiDPI screen.

**Comment by edgardoh** [comment]
@Nilvus , looks the same to me, something else than building it that I should do?

File: data/themes/darktable.css (hunk 1 above)
Comment (Nilvus) [reply data/themes/darktable.css]: pt is normally not recommanded on screens. As css rules are maintain by w3c, see here : https://www.w3.org/Style/Examples/007/units.en.html
But as I don't use HiDPI screen (only full HD one), maybe GTK doesn't respect w3c rules. Could you confirm that ?

Comment (Nilvus) [reply data/themes/darktable.css]: I just read some things on gtk manual. The CSS properties page related to w3c pages. And I also read that HiDPI was added to GTK with using scale factor for HiDPI so I suppose that would be good.

**Comment by Nilvus** [comment]
> @Nilvus , looks the same to me, something else than building it that I should do?

That's not really looks the same to me, but that's not the same as right panel. I've made lot of tests without success. What I propose to you at first yesterday break other dialog box. But I've just had another idea that should much better. See new commit.

**Comment by Nilvus** [comment]
Sorry for two last commit. Someone I forget and I didn't understand on resolving that I was just merging master css... I will stop for tonight.

**Comment by Nilvus** [comment]
Issue #2481 now corrected.

**Comment by Nilvus** [comment]
@edgardoh : could you test last updates of css. What do you think about these changes, especially prefs entries (and filter collection one) as it's related to your issue.

**Comment by TurboGit** [comment]
@Nilvus : sounds like you are correct and I'm wrong:

https://www.w3.org/Style/Examples/007/units.en.html

pt should not be used for screen and px should be preferred.

**Comment by Nilvus** [comment]
By now, I hope this last commit will end this PR. I have just improve contrast with revert back grey_55 to default css color. That make a better contrast especially on filmstrip (difficult to make it better without breaking things).

I also clean grey css as it have redundant lines with darktable.css related theme. A comment needs to be more precise so I updated it.

**Comment by Nilvus** [comment]
@TurboGit : anyway I also found that 1pt=1.33px, so I can't change just the unit. For example, I have replaced font-size of blending tabs from 8pt to 8 px and see that line becoming too small. But changing it to 0.8em (or 10px or 11px, as 8pt = 10.64px) make it better.
By now, I let existing pt as it is (rendering si the most important part of that).

**Comment by Nilvus** [comment]
As I hope UI is better now, see screenshots of new main updates to help review :

**Filter collection entry and arrow dropdown**
filtercollection

**Preferences dialog window**
preferenceswindow

**More condensed darkroom history**
history

**New live sample _(main work by @TurboGit on another commit, just improve text lisibility)_**
livesamples

**More reasonable size of display dropdown arrow on filmic**
filmicarrow

**Comment by edgardoh** [comment]
@Nilvus , much better. Still the controls (text entry, check box) could be smaller (in height), also the space between controls. But if can't be done I can live with it.

**Comment by Nilvus** [comment]
> @Nilvus , much better. Still the controls (text entry, check box) could be smaller (in height), also the space between controls. But if can't be done I can live with it.

You will have to forgot having a smaller height as the window is a grid, I could only reduce height with margin and padding already to 0. And negative values are not recognized. So for that, it's not possible without a Gtk rework...

For space, it's possible by reduce margin from this part :
#preferences_notebook scrolledwindow label,
#preferences_notebook scrolledwindow entry,
#preferences_notebook scrolledwindow button
{
  padding: 0px;
  margin: 5px 0;
}

If I reduce from 5px to 3px, see new spacing (keep quite aerate would be better so I'm not on to reduce spacing more) :

Capture-20190519181316-800x830

**Comment by Nilvus** [comment]
Finally, prefs window is probably better with reducing space as last screen capture. So committed with a darktable-elegant code cleanup (many lines redundant from darktable default css and some lines forgotten and not needed, ie live-sample-data better with darktable default settings).

I also changed font-weight from 400 to normal to have the same setting (400=normal on css) on css.

**Pull request merged**